.eslintcache

```diff
@@ -0,0 +1 @@
+[{"/Users/megumi/Documents/GitHub/Norofff/2nd_year/workflow/workflow-CA/social-media-client/cypress.config.js":"1"},{"size":222,"mtime":1674599867009,"results":"2","hashOfConfig":"3"},{"filePath":"4","messages":"5","suppressedMessages":"6","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"1h6z6k3","/Users/megumi/Documents/GitHub/Norofff/2nd_year/workflow/workflow-CA/social-media-client/cypress.config.js",[],["7","8"],{"ruleId":"9","severity":2,"message":"10","line":6,"column":21,"nodeType":"11","messageId":"12","endLine":6,"endColumn":23,"suppressions":"13"},{"ruleId":"9","severity":2,"message":"14","line":6,"column":25,"nodeType":"11","messageId":"12","endLine":6,"endColumn":31,"suppressions":"15"},"no-unused-vars","'on' is defined but never used.","Identifier","unusedVar",["16"],"'config' is defined but never used.",["17"],{"kind":"18","justification":"19"},{"kind":"18","justification":"19"},"directive",""]
```

Comment:
You can delete, commit then add this file to `gitignore`

Reply:
I was just wondering if this file is something to be ignore. Thanks!